src/index.html

```diff
@@ -7,8 +7,35 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="nav-link">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="logo">
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="list-item"><a href="#" class="nav-link is-active">Apple</a></li>
+          <li class="list-item"><a href="#" class="nav-link">Samsung</a></li>
+          <li class="list-item"><a href="#" class="nav-link">Smartphones</a></li>
+          <li class="list-item">
+            <a
+              href="#"
+              class="nav-link"
+              data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li class="list-item"><a href="#" class="nav-link">Gadgets</a></li>
+          <li class="list-item"><a href="#" class="nav-link">Tablets</a></li>
+          <li class="list-item"><a href="#" class="nav-link">Photo</a></li>
+          <li class="list-item"><a href="#" class="nav-link">Video</a></li>
```

Comment:
```suggestion
          <li class="list-item">
            <a href="#" class="nav-link is-active">Apple</a>
          </li>
          <li class="list-item">
            <a href="#" class="nav-link">Samsung</a>
          </li>
          <li class="list-item">
            <a href="#" class="nav-link">Smartphones</a>
          </li>
          <li class="list-item">
            <a
              href="#"
              class="nav-link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          <li class="list-item">
            <a href="#" class="nav-link">Gadgets</a>
          </li>
          <li class="list-item">
            <a href="#" class="nav-link">Tablets</a>
          </li>
          <li class="list-item">
            <a href="#" class="nav-link">Photo</a>
          </li>
          <li class="list-item">
            <a href="#" class="nav-link">Video</a>
          </li>
```
Consider fixing indents here